src/WebJobs.Script/Description/FunctionInvokerBase.cs

```diff
@@ -156,7 +174,16 @@ private void LogFunctionFailed(FunctionStartedEvent startedEvent, string resultS
                 startedEvent.Success = false;
             }
 
-            TraceWriter.Error($"Function completed ({resultString}, Id={invocationId ?? "0"})");
+            TraceWriter.Error($"Function completed ({resultString}, Id={invocationId ?? "0"})", FunctionTraceProperties);
+        }
+
+        protected TraceWriter CreateUserTraceWriter(TraceWriter traceWriter)
+        {
+            // We create a composite writer to ensure that all user traces get
+            // written to both the original trace writer as well as our file trace writer
+            // This is a "user" trace writer that will mark all traces going through
+            // it as a "user" trace so they are filtered from system logs.
+            return new UserTraceWriter(new[] { traceWriter, FileTraceWriter });
```

Comment:
Since the base trace writer already includes the SystemTraceWriter and Host file writer, I had to introduce the scoped down FileTraceWriter property to avoid duplicate logs.